jcl/src/java.base/share/classes/java/lang/invoke/MemberName.java

```diff
@@ -70,5 +70,8 @@ String getMethodDescriptor() {
 	public Class<?> getDeclaringClass() {
 		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
 	}
+	public boolean isFinal() {
```

Comment:
Should this be wrapped with `Java11` preprocessor tags?

Reply:
Added `Java11` flag.